lib/assert.js

```diff
@@ -271,8 +271,15 @@ function innerDeepEqual(actual, expected, strict, memos) {
       position: 0
     };
   } else {
-    if (memos.actual.has(actual)) {
-      return memos.actual.get(actual) === memos.expected.get(expected);
+    // We prevent up to two map.has(x) calls by directly retrieving the value
+    // and checking for undefined. The map can only contain numbers, so it is
+    // safe to check for undefined only.
+    const expectedMemoA = memos.actual.get(actual);
```

Comment:
Hey @BridgeAR, old PR, I know, but I am trying to understand the implementation in latest master and am confused by this code. Why is this variable called `expectedMemoA`, if it comes from the `actual` Map? Shouldn't it be called `actualMemo`?